

class ExceptionHandler:

    def handle(self, exc):
        return exc

    pass
